Currency Converter

AlgotOlofsson opened the pull request:
Jag har implementerat en currency converter med hjälp utav ett API. Tog även bort css för about_us.

AlgotOlofsson committed:
Currency Converter

Jag har implementerat en currency converter med hjälp utav ett API. Tog även bort css för about_us.

emilaqvist reviewed (changes requested):
Api nyckeln ligger i pushen. Generera ny nyckel, referera till miljövariabel.

AlgotOlofsson committed:
Config

Skapade frontend-config.js för att lagra API-nyckel och lade till filen i .gitignore

AlgotOlofsson commented:
Genererade ny nyckel och refererade till miljövariabel. Finns ingen nyckel i pushen..
